src/components/DataDimension/styles/DropZone.style.js

```diff
@@ -0,0 +1,51 @@
+import css from 'styled-jsx/css'
+
+export default css`
+    .first-dropzone {
+        position: absolute;
+        top: 0;
+        left: 0;
+        z-index: -1;
+        width: 24px;
+        height: 28px;
+        background-color: transparent;
+    }
+
+    .over {
+        z-index: 100;
+    }
+
+    .empty {
+        position: relative;
+        flex-grow: 1;
+        z-index: 100;
+        margin-left: -10px;
+        margin-top: -4px;
+    }
+
+    .over::before,
+    .over::after {
+        content: '';
+        position: absolute;
+    }
+
+    /* the vertical line */
+    .over::before {
+        top: 10px;
+        width: 4px;
+        left: 4px;
+        height: 18px;
+        background-color: #4c9ffe;
```

Comment:
is there a `ui` color for this?

Reply:
No, not right now. But I'll let Joe know. This is the color of the blue drop marker, and there are some blue colors in the design system.

Reply:
Nice catch. Not sure where `#4c9ffe` came from (maybe the original plugin...or past-me) -- anyway, `blue500` can be used instead.